torch/nn/_functions/thnn/activation.py

```diff
@@ -274,6 +327,7 @@ def backward(ctx, grad_output):
         return grad_input, None, None
 
 _all_functions.append(PReLU)
+_all_functions.append(PReLUBackward)
```

Comment:
Can you remove Threshold and LeakyReLU classes above and use auto to create them? They're here only because in this way that they could support double backwards before this refactor